site/ko/blog/new-in-devtools-94/index.md

```diff
@@ -0,0 +1,175 @@
+---
+layout: "layouts/blog-post.njk"
+title: "What's New In DevTools (Chrome 94)"
+authors:
+  - jecelynyeen
+date: 2021-08-24
+updated: 2021-08-24
+description:
+  "Use DevTools in your preferred language, new Nest Hub devices, new CSS container queries badge and more."
+hero: 'image/dPDCek3EhZgLQPGtEG3y0fTn4v82/js1uKEulNBGj0PoJkqZq.jpg'
+alt: ''
+tags:
+  - new-in-devtools
+  - devtools
+  - chrome-94
+---
+
+{% include 'partials/devtools/kr/banner.md' %}
+
+## DevTools 를 이제 여러분들이 선호하는 언어로 사용하세요. {: #localized }
```

Comment:
I notice quite a few title has a dot "." at the end of the title. Is that intentional?